packages/components/src/keyboard-shortcuts/types.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import type { useKeyboardShortcut } from '@wordpress/compose';
+
+type WPKeyboardShortcutConfig = NonNullable<
+	Parameters< typeof useKeyboardShortcut >[ 2 ]
+>;
```

Comment:
A bit weird, but the `WPKeyboardShortcutConfig` type is not directly exported from the wp-compose package. I generally want to avoid doing fragile index-based access on `Parameters<>`, but 🤷 

I've tried to balance readability and DRYness in this types file. Hope it doesn't look too random 😅

Reply:
Yeah, this looks like a good compromise. The only improvement that comes to mind is to add a little inline comment, basically explaining that the reason why we're doing this, is because that type is not exported.

As a small follow-up PR, we may even consider exporting that type from `compose` package and therefore update this line of code too.